packages/mask/src/plugins/FileService/Worker/service.ts

```diff
@@ -1,2 +1,23 @@
-export * from './arweave'
+import * as arweave from './arweave'
+import * as ipfs from './ipfs'
+import * as swarm from './swarm'
+
+const allProviders: any = {
```

Comment:
```suggestion
const allProviders: Record<Provider, ProviderAgent> = {
```

Reply:
sorry, I'm not verfy familiar with typescript